### DISCUSSION TITLE
How to update Dashboard template?

### DISCUSSION THREAD

**Discussion by tomykho** [saadeghi/daisyui#2719] [Q&A] [answered]
I bought Dashboard template from daisyUI store few months ago with version 1.0.2.
How to check if there is any updates? or Do I need to buy new one again just to update the template?

**Comment by saadeghi** [comment, marked as answer]
Thanks for the purchase.
If you click on the link from the email you received, you can download the newest version.

There's no major updates for admin dashboard yet. Just some dependency updates.

Let me know if you have any questions
